src/templates/app/src/{{name}}.cr

```diff
@@ -2,6 +2,9 @@ require "amber"
 require "../config/*"
 require "./models/**"
 require "./mailers/**"
+require "./middleware/**"
```

Comment:
can we rename this to `handlers` instead since is more the nomenclature that Crystal uses?

Reply:
yeah, sounds good.